java_gen/pre-written/pom.xml

```diff
@@ -87,17 +87,17 @@
         <dependency>
             <groupId>io.netty</groupId>
             <artifactId>netty-all</artifactId>
-            <version>4.1.42.Final</version>
+            <version>4.1.50.Final</version>
         </dependency>
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>20.0</version>
+            <version>29.0-jre</version>
         </dependency>
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
-            <version>1.7.25</version>
+            <version>1.7.26</version>
```

Comment:
Latest version of 1.7 series seems to be 1.7.30.
http://www.slf4j.org/news.html
Looking at changes it seems to be minor bug fixes, etc.so it  might make sense to start consuming it everywhere?

Reply:
👍 4419d33